sdk/tests/client/node_api/mqtt.rs

```diff
@@ -53,6 +53,11 @@ async fn test_mqtt() {
             } => {
                 panic!("mqtt disconnected");
             }
+            _ = async {
+                tokio::time::sleep(instant::Duration::from_secs(120)).await;
+            } => {
+                panic!("timed out after 120s");
```

Comment:
We keeping this ?

Reply:
not sure. could maybe use nextest to timeout the whole test instead

Reply:
As you want